src/oop/oop1.py

```diff
@@ -17,3 +17,23 @@
 #     pass
 #
 # Put a comment noting which class is the base class
+class Vehicle:
+    pass
+
+class FlightVehicle(Vehicle): #vehicle
+    pass
+
+class Starship(FlightVehicle): #flightvehicle
+    pass
+
+class GroundVehicle(Vehicle):#Vehicle
+    pass
+
+class Airplane(FlightVehicle):#groundvehicle
```

Comment:
Ground Vehicle is not the base class for Airplane. The comment is asking for which is the parent class.